lib/gpu_memory_service/tests/test_sglang_server_args_compat.py

```
@@ -0,0 +1,124 @@
+
+    assert server_args.enable_memory_saver is True
+
+
+def test_installed_sglang_accepts_memory_saver_after_resolution():
+    server_args = _installed_sglang_server_args()
+
+    _override_server_args(server_args, "dynamo.gms", enable_memory_saver=True)
+
+    try:
+        resolved = _resolved_view(server_args)
+    except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)